docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -316,40 +316,39 @@ Choose your preferred SSH key to use with the cluster, then click Next:
   </figure>
 </p>
 
-Choose *Enabled* to enable the use Amazon EKS and ECR:
+Choose the instance profile role that you have created in Step 2, then click Next:
 
 <p align="center">
   <figure>
     <a  href="../../../assets/images/hopsworksai/aws/eks-hopsworks-create-cluster-1.png">
       <img src="../../../assets/images/hopsworksai/aws/eks-hopsworks-create-cluster-1.png" alt="Choose Enabled">
```

Comment:
This is not the screenshot for the instance profile

Reply:
Fixed